src/main/java/edu/iis/powp/adapter/MyAdapter.java

```diff
@@ -1,37 +1,38 @@
 package edu.iis.powp.adapter;
- import edu.iis.client.plottermagic.IPlotter;
+
+import edu.iis.client.plottermagic.IPlotter;
 import edu.kis.powp.drawer.panel.DrawPanelController;
 import edu.kis.powp.drawer.shape.ILine;
 import edu.kis.powp.drawer.shape.LineFactory;
- /**
- * Plotter adapter to drawer with several bugs. 
+
+/**
+ * Plotter adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements IPlotter
-{ 
+public class MyAdapter extends DrawPanelController implements IPlotter {
 	private int startX = 0, startY = 0;
-	
-    public MyAdapter() {
+
+	public MyAdapter() {
 		super();
 	}
-    
+
 	@Override
-    public void setPosition(int x, int y)
-    {
-        this.startX = x;
-        this.startY = y;
-    }
-     @Override
-    public void drawTo(int x, int y)
-    {
-        ILine line = LineFactory.getBasicLine();
-    	line.setStartCoordinates(this.startX, this.startY);
-        line.setEndCoordinates(x, y);
+	public void setPosition(int x, int y) {
+		this.startX = x;

```

Comment:
Why so many changes to code formatting?
It makes tracking actual changes more difficult.